JS/Exercise_25/index.js

```diff
@@ -0,0 +1,52 @@
+let weekdayNumber = 7;
+
+if (weekdayNumber == 1) {
+    console.log('Today is Monday');
+} else if (weekdayNumber == 2) {
+    console.log('Today is Tuesday');
+} else if (weekdayNumber == 3) {
+    console.log('Today is Wednesday');
+} else if (weekdayNumber == 4) {
+    console.log('Today is Thursday');
+} else if (weekdayNumber == 5) {
+    console.log('Today is Friday');
+} else if (weekdayNumber == 6) {
+    console.log('Today is Saturday');
+} else if (weekdayNumber == 7) {
+    console.log('Today is Sunday');
+}
+
```

Comment:
define the variable on top and set the value in cases

Reply:
Sorry, what value? I thought we were supposed to use 'else if' first before using case switch?